app/_data/docs_nav_kgo_1.2.x.yml

```diff
@@ -54,8 +54,6 @@ items:
     items:
       - text: Overview
         url: /production/
-      - text: Install
```

Comment:
We should keep the standalone install page. It's one that many people look for when visiting for the first time

Reply:
Got it, reverted that change.